tests/validation/tests/v3_api/test_hosted_eks_cluster.py

```diff
@@ -173,8 +174,10 @@ def test_eks_v2_create_import_cluster():
     """
     Create an imported EKS cluster with some default values in EKS config
     """
-    display_name = create_resources_eks()
-    cluster_name = random_test_name("test-auto-eks")
+    # Temporarily commented, as botoclient is not working
+    # display_name = create_resources_eks()
+    # cluster_name = random_test_name("test-auto-eks")
+    cluster_name = MANUAL_IMPORTED_EKS_CLUSTER
```

Comment:
Can we add an if condition - if MANUAL_IMPORTED_EKS_CLUSTER then use that, else create a cluster?

Reply:
Added check for MANUAL_IMPORTED_EKS_CLUSTER. If name is not found, cluster will be created using botoclient.